Makefile

```diff
@@ -31,6 +34,10 @@ endif
 # Install all necessary dependencies.
 deps: $(distro_deps)
 
+UNAME_S := $(shell uname -s)
+ifeq ($(UNAME_S),Darwin)
```

Comment:
I am not totally sure about this part, whether this works correct on MacOS or not?

Reply:
Just checked on an OS X system:

```
$ uname -s
Darwin
```

So I think it should work fine.